.github/actions/pack-and-deploy/action.yml

```diff
@@ -9,8 +9,13 @@ runs:
         # on it: app-tap/gh-4983-tnt-e-assert-false-hangs.test.lua.
         export PACKPACK_EXTRA_DOCKER_RUN_PARAMS='--init'
 
-        # Create packages.
-        make -f .pack.mk package
+        # Create packages. For versions 3.x creates static packages.
+        # For old versions 1.x and 2.x creates packages of specific OS.
+        if ${{ (startsWith(github.ref, 'refs/tags/3.') }}; then
```

Comment:
There is a typo. Please remove `(` before `startsWith`.

Reply:
outdated